packages/svelte/src/hooks/useAccount.svelte.ts

```diff
@@ -0,0 +1,40 @@
+import {
+  type Config,
+  type GetAccountReturnType,
+  type ResolvedRegister,
+  getAccount,
+  watchAccount,
+} from '@wagmi/core'
+import type {
+  ConfigParameter,
+  RuneParameters,
+  RuneReturnType,
+} from '../types.js'
+import { useConfig } from './useConfig.svelte.js'
+
+export type UseAccountParameters<config extends Config = Config> =
+  RuneParameters<ConfigParameter<config>>
+
+export type UseAccountReturnType<config extends Config = Config> =
+  RuneReturnType<GetAccountReturnType<config>>
+
+/** https://wagmi.sh/react/api/hooks/useAccount */
+export function useAccount<config extends Config = ResolvedRegister['config']>(
```

Comment:
Is useFoo normal for svelte apps? Would prefer it to just be `getAccount`

Reply:
Noticing tanstack docs does this pattern of useFoo
![image](https://github.com/user-attachments/assets/2295bee4-0e2f-4e27-9d6a-89c65915dd3c)